storybook/stories/playground/zoom-out/pattern.js

```diff
@@ -0,0 +1,39 @@
+export const pattern = `<!-- wp:cover {"customOverlayColor":"#eb4c77","contentPosition":"center center","isDark":false,"align":"full","style":{"spacing":{"padding":{"top":"6vw","right":"6vw","bottom":"6vw","left":"6vw"},"margin":{"top":"0"}}}} -->
```

Comment:
This is just a random pattern I took from the pattern library to show case the zoom-out in storybook.